testnet/stacks-node/src/mockamoto.rs

```diff
@@ -0,0 +1,565 @@
+            network_epoch: PEER_VERSION_EPOCH_2_1
+        },
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch22,
+            start_height: 3,
+            end_height: 4,
+            block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
+            network_epoch: PEER_VERSION_EPOCH_2_2
+        },
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch23,
+            start_height: 4,
+            end_height: 5,
+            block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
+            network_epoch: PEER_VERSION_EPOCH_2_3
+        },
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch24,
+            start_height: 5,
+            end_height: 6,
+            block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
+            network_epoch: PEER_VERSION_EPOCH_2_4
+        },
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch30,
+            start_height: 6,
+            end_height: STACKS_EPOCH_MAX,
+            block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
+            network_epoch: PEER_VERSION_EPOCH_2_4
+        },
+    ];
+}
+
+fn make_snapshot(
+    parent_snapshot: &BlockSnapshot,
+    miner_pkh: &Hash160,
+) -> Result<BlockSnapshot, ChainstateError> {
+    let burn_height = parent_snapshot.block_height + 1;
+    let mut mock_burn_hash_contents = [0u8; 32];
+    mock_burn_hash_contents[0..8].copy_from_slice((burn_height + 1).to_be_bytes().as_ref());
+    let mut mock_consensus_hash_contents = [0u8; 20];
+    mock_consensus_hash_contents[0..8].copy_from_slice((burn_height + 1).to_be_bytes().as_ref());
+
+    let new_bhh = BurnchainHeaderHash(mock_burn_hash_contents);
+    let new_ch = ConsensusHash(mock_consensus_hash_contents);
+    let new_sh = SortitionHash([1; 32]);
+
+    let new_snapshot = BlockSnapshot {
+        block_height: burn_height,
+        burn_header_timestamp: 100 * u64::from(burn_height + 1),
+        burn_header_hash: new_bhh.clone(),
+        parent_burn_header_hash: parent_snapshot.burn_header_hash.clone(),
+        consensus_hash: new_ch.clone(),
+        ops_hash: OpsHash([0; 32]),
+        total_burn: 10,
+        sortition: true,
+        sortition_hash: new_sh,
+        winning_block_txid: Txid([0; 32]),
+        winning_stacks_block_hash: BlockHeaderHash([0; 32]),
+        index_root: TrieHash([0; 32]),
+        num_sortitions: parent_snapshot.num_sortitions + 1,
+        stacks_block_accepted: true,
+        stacks_block_height: 1,
+        arrival_index: parent_snapshot.arrival_index + 1,
+        canonical_stacks_tip_height: 1,
+        canonical_stacks_tip_hash: BlockHeaderHash([0; 32]),
+        canonical_stacks_tip_consensus_hash: new_ch.clone(),
+        sortition_id: SortitionId::new(&new_bhh.clone(), &PoxId::new(vec![true])),
+        parent_sortition_id: parent_snapshot.sortition_id.clone(),
+        pox_valid: true,
+        accumulated_coinbase_ustx: 0,
+        miner_pk_hash: Some(miner_pkh.clone()),
+    };
+    Ok(new_snapshot)
+}
+
+pub struct MockamotoNode {
```

Comment:
Can you add a comment on this struct, or a top-level commenting explaining what Mockamoto does and why it is useful? It doesn't need to be anything to complicated, since this is just temporary, but I think this will be helpful for others using it during development.